libs/design-system/src/atoms/inputs/textAreaInputs.tsx

```diff
@@ -53,7 +53,7 @@ export const TextAreaInput = ({
       id={inputId}
       name={inputId}
       className={cn(
-        'plasmo-antialiased plasmo-text-gray plasmo-border plasmo-border-solid plasmo-border-gray plasmo-rounded-md plasmo-w-full plasmo-p-2 focus:plasmo-outline-none focus:plasmo-border-cyan',
+        'plasmo-antialiased plasmo-resize-none plasmo-text-grey-light plasmo-bg-grey-transparent plasmo-border plasmo-border-solid plasmo-border-grey-transparent plasmo-rounded-md plasmo-w-full plasmo-p-2 focus:plasmo-outline-none focus:plasmo-border-main',
```

Comment:
Why no resize?

Reply:
I find them messy, especially in an already narrow side panel.

Whatever is written in this text area's supposed to end up in a text *property* field in a Notion db. It's **not a place for long text blocks**. I don't want the resizable text area to nudge users in the wrong direction.

That being said, it's not a hard opinion. My mind can be changed!